src/login/lobby.cpp

```diff
@@ -343,7 +343,23 @@ int32 lobbydata_parse(int32 fd)
 
                     ShowInfo("lobbydata_parse: zoneid:(%u),zoneip:(%s),zoneport:(%u) for char:(%u)", ZoneID, ip2str(ntohl(ZoneIP)), ZonePort, charid);
 
-                    if (maint_config.maint_mode == 0 || gmlevel > 0)
+                    // Check the number of sessions
+                    uint16 SessionCount = 0;
+
+                    fmtQuery = "SELECT COUNT(client_addr) \
+                                FROM accounts_sessions \
+                                WHERE client_addr = %u;";
+                    if (sql->Query(fmtQuery, sd->client_addr) != SQL_ERROR && sql->NumRows() != 0)
+                    {
+                        sql->NextRow();
+                        SessionCount = (uint16)sql->GetIntData(0);
+                        if (login_config.login_limit != 0 && SessionCount >= login_config.login_limit)
+                        {
+                            ShowWarning("Already %u active sessions for %u (Limit is %u)", SessionCount, sd->client_addr, login_config.login_limit);
```

Comment:
Could you retrieve and print the account name here, either by a lookup (if we have the data locally), or by adding a join to your query.

Reply:
![image](https://user-images.githubusercontent.com/651924/173628212-01a260dc-02fc-4611-8600-8ad43c55d017.png)

no juice for cacaroon